server/commands/cycle.js

```diff
@@ -1,47 +1,65 @@
 /* global graphQLFetcher, notifyUser, commandsConfig */
 
-function invokeLaunchCycleAPI(lgJWT) {
+function invokeUpdateCycleStateAPI(state, lgJWT) {
   const baseURL = process.env.NODE_ENV === 'development' ? 'http://game.learnersguild.dev' : 'https://game.learnersguild.org'
   const mutation = {
-    query: 'mutation { launchCycle { id } }'
+    query: 'mutation($state: String!) { updateCycleState(state: $state) { id } }',
+    variables: {state},
   }
   return graphQLFetcher(lgJWT, baseURL)(mutation)
     .then(data => data.launchCycle)
 }
 
-function launchCycle(commandInfo) {
+function handleUpdateCycleStateCommand(commandInfo, state, msg) {
   try {
     const {lgJWT, lgPlayer} = Meteor.user().services.lgSSO
     if (!lgJWT || !lgPlayer) {
       throw new Error('You are not a player in the game.')
```

Comment:
This isn't right. You don't need to be a player in the game, you need to be a moderator. See my other comment above about `lgUser` vs. `lgPlayer` on the Meteor user record when destructuring.